app/diagnostics_test.go

```diff
@@ -68,7 +68,9 @@ func TestDiagnostics(t *testing.T) {
 		t.SkipNow()
 	}
 
-	th := Setup(t)
+	th := SetupWithCustomConfig(t, func(config *model.Config) {
```

Comment:
It's important that plugins be /disabled/ before the server starts, since shutting down plugins doesn't set environment back to `nil`.